monitoring/monitors/viya/serviceMonitor-sas-opensearch.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: sas-opensearch
+spec:
+  endpoints:
+  - basicAuth:
+      password:
+        key: password
+        name: sas-opendistro-sasadmin-secret
+      username:
+        key: username
+        name: sas-opendistro-sasadmin-secret
```

Comment:
Adding **port: https** here (after line13, indentation matching metricRelabelings and basicAuth) appears to limit the Prometheus scraping to only the desired port.